labs.acousticbrainz.org/_config.yml

```diff
@@ -7,4 +7,4 @@ url: "http://acousticbrainz.org"
 
 # Build settings
 markdown: kramdown
-permalink: /:title
+permalink: /:title.html
```

Comment:
it looks like what was missing here is a trailing `/` to make it correctly build the directory structure:

    permalink: /:title/

I'd prefer that we keep with this directory structure as it's the same as what we've historically had

Reply:
done.